File: api-java/src/main/java/fr/huiitre/tools/modules/riot/valorant/application/catalog/view/ValorantStoreBundle.java
package fr.huiitre.tools.modules.riot.valorant.application.catalog.view;

import java.util.List;

public record ValorantStoreBundle(
    String assetId,
    String name,
    String bannerUrl,
    List<ValorantStoreOffer> items,
    int totalBaseCost,
    int totalDiscountedCost,
    int discountPercent,
    long remainingSeconds
) {}

File: api/Modules/Riot/Valorant/Application/Catalog/Views/ValorantStoreBundle.cs
namespace Tools.Api.Modules.Riot.Valorant.Application.Catalog.Views;

public sealed record ValorantStoreBundle(
    string AssetId,
    string Name,
    string BannerUrl,
    List<ValorantStoreOffer> Items,
    int TotalBaseCost,
    int TotalDiscountedCost,
    int DiscountPercent,
    long RemainingSeconds
);